projects/demo/src/modules/charts/line-chart/examples/1/index.ts

```diff
@@ -11,7 +12,7 @@ import {TUI_DEFAULT_STRINGIFY} from '@taiga-ui/cdk';
     encapsulation,
 })
 export class TuiLineChartExample1 {
-    readonly value = [
+    readonly value: ReadonlyArray<TuiPoint> = [
```

Comment:
`readonly TuiPoint[]` is a more modern way to write this :) Let's use it. I tend to use `ReadonlyArray` only for two dimensional arrays, not a fan of `readonly (readonly string[])[]` — hard to read imho

Reply:
fixed